CMake/cdat_modules_extra/cdat_common_environment.cmake.in

```diff
@@ -1,5 +1,5 @@
 message("[INFO] ADDITIONAL CFLAGS ${ADDITIONAL_CFLAGS}")
-set(ENV{PATH} "@cdat_EXTERNALS@/bin:$ENV{PATH}")
+set(ENV{PATH} "@SB_BIN_DIR@:@cdat_EXTERNALS@/bin:$ENV{PATH}")
```

Comment:
@aashish24 I'm not clear about this? You don't like the name? Or you don't like that it is in the path? It has to be in the path in order to bypass the brew installed `cython`